cookbook/deployment/workflow/lp_schedules.py

```diff
@@ -27,8 +29,11 @@ def date_formatter_wf(kickoff_time: datetime):
 
 
 # %%
-# Cron Expression
-# ===============
+# There are two types of schedules: Cron Schedules and Fixed Rate Intervals.
```

Comment:
```suggestion
# The `date_formatter_wf` workflow can be scheduled either using Cron Schedules or Fixed Rate Intervals.
```

Reply:
can we lower case and just say "using cron schedules or fixed intervals."?